tests/integration/components/new-stepper-test.js

```
@@ -1,38 +1,255 @@
 import { module, test } from 'qunit';
 import { setupRenderingTest } from 'website-www/tests/helpers';
-import { render } from '@ember/test-helpers';
+import { render, click } from '@ember/test-helpers';
 import { hbs } from 'ember-cli-htmlbars';
+import sinon from 'sinon';
+import Service from '@ember/service';
+import { STEP_DATA_STORAGE_KEY } from 'website-www/constants/new-join-form';
+
+function clearAllStepData() {
+  Object.values(STEP_DATA_STORAGE_KEY).forEach((key) =>
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)